dateparser/languages/language.py

```diff
@@ -24,40 +26,51 @@ def __init__(self, shortname, language_info):
             if isinstance(value, int):
                 simplification[key] = str(value)
 
+        self._cached = self
```

Comment:
If we would customize settings before importing `parse`, then it would work alright. But if we'd customize settings after importing `parse`, it won't work hence requiring generation of languages again.